src/main/resources/assets/gregtech/lang/en_us.lang

```diff
@@ -4701,6 +4701,7 @@ behaviour.paintspray.red.tooltip=Can paint things in Red
 behaviour.paintspray.black.tooltip=Can paint things in Black
 behaviour.paintspray.uses=Remaining Uses: %,d
 behaviour.paintspray.offhand=Hold in offhand to color while placing blocks
+behaviour.paintspray.crouch=Crouch to spray a network of pipes
```

Comment:
This should say `Crouch to spray connected pipes` instead. Using "network" isn't a very user-friendly term here.